lib/NetPacks.h

```diff
@@ -1620,7 +1621,6 @@ struct BattleStackAttacked
 		h & flags;
 		h & killedAmount;
 		h & damageAmount;
-		h & effect;

```

Comment:
For 1.2 serialization is not broken yet. Can be fixed easily
if(version < xxx)
{
  int stub;
  h & stub;
}

Reply:
There were somewhat large changes for netpacks, so no. This case may be fixable, but others probably won't be. 
So multiplayer compatibility IS broken. Save compatibility is doable, will recheck